content/applications/general/users.rst

```diff
@@ -4,143 +4,235 @@
-form. Click on the *Action* button and select *Change Password*.
+form. Click on the :guilabel:`⚙️ Actions` icon and select :guilabel:`Change Password`. Enter a new
+password and confirm by clicking on :guilabel:`Change Password`.
 
 .. image:: users/change-password.png
-   :alt: Change another user's password on Odoo
-
-Enter a new password and confirm by clicking on *Change Password*.
+   :align: center
+   :alt: Change a user's password on Odoo.
 
 .. note::
    This operation only modifies the password of the users locally and does not affect their odoo.com
-   account. If you want to change the odoo.com password, you can :ref:`send the password reset
-   instructions <users/reset-password-email>`.
+   account. If the odoo.com password needs to be changed, then use the :ref:`send the password reset
+   instructions <users/reset-password-email>`. Odoo.com passwords will grant access to the *My
+   Databases* page and other portal features.
 
-Click on *Change Password* one more time. You are then redirected to an Odoo login page where you
-can reaccess your database using your new password.
+After clicking on :guilabel:`Change Password`, the page is redirected to an Odoo login page where
+the database can be re-accessed using the new password.
 
 .. _users/multi-companies:
 
 Multi Companies
 ===============
 
-The *Multi Companies* field allows you to set to which of the multiple companies database you hold
-the user can have access.
+The :guilabel:`Multi Companies` field allows an administrator to allow access to multiple companies
+for existing users. To configure a multi-company environment for a user, navigate to the user by
+going to: :menuselection:`Settings app --> Users --> Manage users`. Then select the user to
+configure with multi-company access.
 
-.. note::
-   Note that if not handled correctly, it may be the source of a lot of inconsistent multi-company
-   behaviors. Therefore, a good knowledge of Odoo is required. For technical explanations refer
-   to :doc:`this </developer/howtos/company>` documentation.
+Under :guilabel:`Multi Companies` set the fields labeled :guilabel:`Allowed Companies` and
+:guilabel:`Default Company`.
+
+The :guilabel:`Allowed Companies` field can contain multiply companies. These will be the companies
+the user has access to, and can make edits, according to the set access rights. The
+:guilabel:`Default Company` is the company that the user will default to upon logging in each time.
+This field can contain one company.
+
+.. warning::
+   Note that if the multi-company access is not handled correctly, it may be the source of a lot of
+   inconsistent multi-company behaviors. Therefore, experienced Odoo users should only make access
+   rights changes to users for multi-company configuration. For technical explanations refer to the
+   developer documentation on :doc:`../../../developer/howtos/company`.
```

Comment:
```suggestion
   If multi-company access is not configured correctly, it could lead to
   inconsistent multi-company behaviors. Because of this, only experienced Odoo users should make access
   rights changes to users for databases with a multi-company configuration. For technical explanations, refer to the
   developer documentation on :doc:`../../../developer/howtos/company`.
```